src/main/java/org/jmhsrobotics/frc2025/Constants.java

```diff
@@ -48,6 +48,26 @@ public static class ElevatatorConstants {
     public static final double kP = .01;
     public static final double kI = .00;
     public static final double kD = .00;
+    public static final double kHeightTolerance = 0.1;
+  }
+
+  public static class WristConstants {
+    public static final int kMotorId = 40;
+
+    public static final double kRotationIntakeCoral = 0;
+
+    public static final double kRotationLevel1 = 10;
+    public static final double kRotationLevel2 = 25;
+    public static final double kRotationLevel3 = 25;
+    public static final double kRotationLevel4 = 75;
+
+    public static final double kRotationAlgae = 190;
+    public static final double kRotationProcesser = 180;
+
```

Comment:
Make sure to include units.
IE kRotationL1Degrees etc.
Same with Heights (Meters)